package.json

```diff
@@ -1,44 +1,39 @@
 {
-  "name": "react-native-heap-analytics",
-  "version": "3.0.0",
-  "description": "React Native wrapper for Heap Analytics.",
+  "name": "@heap/react-native-heap",
+  "version": "0.1.0-beta.1",
+  "description": "React Native event tracking with Heap.",
   "license": "MIT",
-  "author": "Andy Isaacson <andyi@heapanalytics.com>",
-  "homepage": "https://github.com/heap/react-native-heap-analytics",
-  "repository": "heap/react-native-heap-analytics",
+  "author": "Heap <http://www.heapanalytics.com>",
+  "homepage": "https://github.com/heap/react-native-heap",
+  "repository": "heap/react-native-heap",
   "keywords": [
     "analytics",
     "heap",
     "ios",
-    "jolt",
+    "android",
     "react-native"
   ],
   "main": "js/Heap",
   "files": [
     "android",
     "ios",
     "js",
-    "react-native-heap-analytics.podspec"
+    "react-native-heap.podspec"
   ],
+  "publishConfig": {
+    "access": "public"
+  },
   "dependencies": {
     "flat": "^4.1.0",
     "react-native-package": "^2.5.0"
   },
   "peerDependencies": {
-    "react-native": "*"
+    "react-native": "x.x"
```

Comment:
My npm foo is weak. Why do we need this line?

Reply:
We don't _really_ need the line I don't think.  This would issue an error if you try to use this package in a project where you didn't install React Native alongside it.  In other words, you're doing something very wrong!

Using `x.x` is valid syntax for _any_ version of React Native, whereas the `*` is not.